chore(AmplifyJS): update the doc of Next.js dx (#6210)
aws-amplify/docs: src/pages/lib/ssr/nextjs/q/platform/[platform].mdx

### Configure Amplify for Server Side Usage

To use Amplify APIs on the server-side of your Next.js app, you will need to
create a `runWithAmplifyServerContextRunner` function.

> You can create an `amplifyServerRunner.ts` file under `src` in your codebase.

Reviewer: Reason for edit: new Next.js apps discourages use of "src/"
```suggestion
You can create an `amplifyServerRunner.ts` file under a `utils` folder in your codebase.
```

In this file, you will import the Amplify configuration object from the
`amplifyconfiguration.json` file that is generated by the Amplify CLI, and use
the `createServerRunner` function to create the `runWithAmplifyServerContextRunner`
function.

For example, the `amplifyServerRunner.ts` file may contain the following content:

```typescript
import { createServerRunner } from "@aws-amplify/adapter-nextjs";

> import config from './amplifyconfiguration.json';

Reviewer: uses aliasing
```suggestion
import config from '@/amplifyconfiguration.json';
```

```

`layout.tsx`:

> ```jsx

Reviewer: does `tsx` not work?

Author: `tsx` doesn't apply any syntax highlighting :/

routes. For example:

```typescript
import { fetchAuthSession } from 'aws-amplify/auth/server';
import { NextRequest, NextResponse } from 'next/server';

> import { runWithAmplifyServerContext } from './amplifyServerRunner';

Reviewer: ```suggestion
import { runWithAmplifyServerContext } from './utils/amplifyServerRunner';
```
